models/match_detail.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from datetime import datetime
+import time
+
+
+class MatchDetails:
+    def __init__(self, dict):
+        self.matchId = dict.get('metadata', {}).get('matchId')
+        self.gameCreation = (datetime.fromtimestamp(dict.get('info', {}).get('gameCreation') // 1000)).strftime("%m-%d %I:%M %p")
+        self.gameDuration = time.strftime('%H:%M:%S',time.gmtime(dict.get('info', {}).get('gameDuration')))
+        self.gameMode = dict.get('info', {}).get('gameMode')
+        self.puuid = dict.get("puuid")
+
+    def merge(self, dict):
+        return dict.update(self)
+
+class ParticipantMatchDetails:
```

Comment:
Is there a reason these aren't in separate files? This seems like you are defining two classes and then combining them back down into one class. Not sure if I follow that entirely, but worth explaining why this was the approach you took (either here or in comments)

Reply:
I am not a programmer and thought it was ok to have 2 classes in the same py file.

I think they belong on the same page as the 1st class grabs match specific details and creates an object.

Then the 2nd class grabs participant specific details and creates an object.

Since each dict of the matches_dict being sent back to Index.py should have all the details for 1 match to display on index.html. Combining them back into one class is the right solution? 

This is where my lack of experience and best practice  knowledge get me educated, but it does work. Now to learn how the "industry" would handle this!

Reply:
I am legitimately not sure if this is an issue or not. If I were looking at this from a Java perspective, this would be very weird because of "strict typing." From what I understand, you're smashing 2 different classes together and getting 1 class out of it... This may be "correct" for Python but I am legitimately not sure. Let's leave it as is and if it causes problems later we can backtrack